### PR TITLE
Allow sudo and thefuck plugins to be loaded before vi-mode

### LINKED ISSUE #7431: Using vi-mode plugin conflicts with sudo plugin's EscEsc sequence
Any chance any of these plugins can be modified so they do not conflict with each other's functionality? 
For instance, configurable binding for **sudo** plugin's **Esc Esc** sequence would be a decent solution I guess.

### PR DESCRIPTION
**Pull request by SomeDer**
fixes #7431

### CODE REVIEW

**Comment by mcornella** [comment]
A better approach is to define the key binding for vicmd and viins keymap in the sudo plugin: replace the bindkey line in the sudo plugin for these three:
```zsh
bindkey -M emacs '\e\e' sudo-command-line
bindkey -M vicmd '\e\e' sudo-command-line
bindkey -M viins '\e\e' sudo-command-line
```

**Comment by SomeDer** [comment]
@mcornella I've copied those lines there and also to `thefuck` which had the same problem.

File: plugins/thefuck/thefuck.plugin.zsh [hunk 1]
```diff
@@ -15,4 +15,6 @@ fuck-command-line() {
 }
 zle -N fuck-command-line
 # Defined shortcut keys: [Esc] [Esc]
-bindkey "\e\e" fuck-command-line
+bindkey -M emacs '\e\e' sudo-command-line
```
Comment (mcornella) [inline plugins/thefuck/thefuck.plugin.zsh]: This should read `fuck-command-line`

Comment (SomeDer) [reply plugins/thefuck/thefuck.plugin.zsh]: OK, I've fixed this.

**Pull request merged**